syft/execution/translation/torchscript.py

```diff
@@ -0,0 +1,36 @@
+from torch import jit
+from syft.execution.placeholder import PlaceHolder
+from syft.execution.translation.abstract import AbstractPlanTranslator
+
+
+class PlanTranslatorTorchscript(AbstractPlanTranslator):
+    """Performs translation from 'list of ops' Plan into torchscript Plan"""
+
+    def __init__(self, plan):
+        super().__init__(plan)
+
+    def translate(self):
+        plan = self.plan
+
+        args_shape = plan.get_args_shape()
+        args = PlaceHolder.create_placeholders(args_shape)
+
+        # Temporarily remove reference to original function
```

Comment:
To force `Plan.__call__` to use recorded operations instead of original function:
https://github.com/OpenMined/PySyft/blob/vvm/torchscript-plan/syft/execution/plan.py#L236

Reply:
Yep, but does it mean that `jit.trace` doesn't work if we execute the Plan with forward()?
I mean, the behavior should be the same. Or maybe this _should_ is the problem